custom-http/src/main/java/com/moneytransfer/core/ktor/model/Accounts.kt

```diff
@@ -0,0 +1,19 @@
+package com.moneytransfer.core.ktor.model
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Accounts(val number: String, val contents: List<AccountsItem>)
+
+@Serializable
+data class AccountsItem(val name: String, val bank: String)
+
+
+val accountsStorage = listOf(Accounts(
+    "11234567", listOf(
+        AccountsItem("Name1", "Bank1")
+    )),
+    Accounts("12345678", listOf(
+        AccountsItem("Name2", "Bank2")
+    ))
+)
```

Comment:
I think it better if you can add data here that is provided us.

Reply:
Sure! will do that.